docs/src/user/manual/user-roles.md

```diff
@@ -17,6 +15,178 @@ There are currently 10 roles:
+
+    * **Access with conditions**: The provincial data scientist can access the following pages with conditions.
+
+        1. **Organisation member page**: Can only add organisation members if instated as a manager of an organisation/ their own organisation.
+
+        2. **Explore page map tab**: Can see all data on the map at property level fro their region and filter by organisation, year and species etc.
+
+        3. **Explore page reports tab**: Can see all data on the sawps platform for all properties to property level, can filter by organisation, property and species, at a regional scale.
+
+        4. **Explore page charts tab**: Can see all data on the sawps platform for all properties to property level, can filter by organisation, property and species, at a regional scale.
+
+        5. **Data upload**: Can upload data if they are part of an organisation.
+
+        6. **Property upload**: Can upload property if they are part of an organisation.
+
+        7. **Species upload**: Can upload species if they are part of an organisation.
+
+        8. **Automated reporting**: All types of report available for regions as per filtering by user.
+
+        9. **Download csv of data from filtered table**: Can download all types of report available for regions as per filtering by user.
+
+        10. **Download charts image**: Can download all types of charts available for regions as per filtering by user.
+
+8. **National data scientist**: A national data reviewer that can view data down to property lever and use the data for e.g. planning and permitting.
+
+    ![National data scientist](./img/user-role-8.png)
+
+    * **Full access**: The national data scientist have full access to login, register, about, contact, profile, notification, reminders, two-factor authentication, and help page. National data scientist can access documentation site.
+
+    * **Access with conditions**: The national data scientist can access the following pages with conditions.
+
+        1. **Organisation member page**: Can only add organisation members if instated as a manager of an organisation/ their own organisation.
+
+        2. **Explore page map tab**: Can see all data on the map at property level for South Africa and filter by organisation, year and species etc.
+
+        3. **Explore page reports tab**: Can see all data on the sawps platform for all properties to property level, can filter by organisation, property and species, for all of South Africa.
+
+        4. **Explore page charts tab**: Can see all data on the sawps platform for all properties to property level, can filter by organisation, property and species, for all of South Africa.
+
+        5. **Data upload**: Can upload data if they are part of an organisation.
```

Comment:
Either have a category called can upload data and explain that this includes property and species data or just have property and species upload instead because having data as a category of its own but then putting the subcategories on their own may be misleading. What's your take on this @Jeremy-Prior ?

Reply:
Apart from the property and species upload, is there another data upload category?

If not, then yes, this is misleading and superfluous. It would be better to have a single point of `data upload` and then explain that the property and species upload is part of that.

Reply:
yes they are the only type of data uploads

Reply:
Then I say have one `data upload` point mentioning the  property and species upload.